src/drivers/github.js

```diff
@@ -170,7 +170,7 @@ class Github {
         await fs.unlink(runnerCfg);
       } catch (e) {
         const arch = process.platform === 'darwin' ? 'osx-x64' : 'linux-x64';
-        const ver = '2.274.2';
+        const ver = '2.278.0';
```

Comment:
?

Should we be auto-checking this?

Reply:
Either that or always using the latest version; loosely related to #615.

Reply:
Good catch. I think that this come because I was actually checking something else.
 > Either that or always using the latest version, though that feels a bit risky;
 
Thats the thing we have to update it from time to time because what the runner does is autoupdate itself! So we download the old version and before starting the runner autoupdates loosing more time 🤦

Reply:
🤦🏼  That's different enough for a separate issue, but it would be nice to automatically determine the latest version on download, instead of hardcoding it.

Reply:
> 🤦🏼 That's different enough for a separate issue,

Thats why I said 

> Good catch. I think that this come because I was actually checking something else in another PR

Said that it wont hurt here since actually the runner do it

Reply:
Opened #641